.travis.yml

```diff
@@ -180,10 +199,11 @@ jobs:
       env:
         - azure-android
         - maven-android
+        - GO111MODULE=on
       git:
         submodules: false # avoid cloning ethereum/tests
       before_install:
-        - curl https://dl.google.com/go/go1.13.6.linux-amd64.tar.gz | tar -xz
+        - curl https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz | tar -xz
```

Comment:
It seems like this is split between `1.14.11` and `1.14.2` We don't use travis so this is fine, but it's also referenced in appveyor.

Reply:
Do we use appveyor?  If so, I will update it there too.  For the travis config I figured better to leave it since we don't use it so there is no need to diverge on that from upstream.

Reply:
Looking close I don't think so. It looks like upstream uses appveyor for windows builds which we don't have configured